libraries/ESP32/examples/DeepSleep/TouchWakeUp/TouchWakeUp.ino

```diff
@@ -16,8 +16,10 @@ Pranav Cherukupalli <cherukupallip@gmail.com>
 
 #if CONFIG_IDF_TARGET_ESP32
```

Comment:
shouldn't this be based on touch version, rather than chip?

Reply:
Can be. Will do